### DISCUSSION TITLE
MacOS 11 (Big Sur) - Save dialog not appearing

### DISCUSSION THREAD

**Discussion by KubaInAir** [airsdk/Adobe-Runtime-Support#721] [Q&A]
We have an Adobe Air app that is supposed export files and save.  This was working well on OSX 10.x, but this functionality stopped working on MacOS 11.   Is this an issue others are experiencing?    Any suggestions for how to address this?

**Comment by ajwfrost** [comment]
Hi

Which version of AIR are you using there? Is this with the "shared runtime" from Harman, or from Adobe, or with an application that includes the "captive runtime"?

(FYI there is an issue in the "shared runtime" currently available form Harman related to permission management, which should be updated soon...)

thanks

**Reply by KubaInAir** [reply]
I'm using the latest version (33.1) of Air by Harman runtime.    I installed the runtime on OSX 10.15 and the app works correctly, the save dialog shows up.   On Big Sur the app starts and mostly runs, but the save dialog to export the file doesn't appear.

**Reply by ajwfrost** [reply]
Okay thanks - so yes this is a bug in that runtime, we're aiming to get a new version uploaded within the next two weeks that would fix this.

**Reply by twhamilton** [reply]
Hello, we are experiencing a problem with the flash.filesystem.file.browseForDirectory that showed up with the Big Sur release. It works on the first call then shows an empty white dialog on subsequent calls, until the app is restarted. 
Was wondering if this is related to the Save dialog fix mentioned here and if it will be addressed in the release?
We are using captive runtime installs. AIR version 32
